flink/metadata.csv

```diff
@@ -0,0 +1,89 @@
+flink.taskmanager.Status.JVM.Memory.Direct.Count,gauge,,,,The number of buffers in the direct buffer pool,0,flink,taskmanager direct buffers
+flink.taskmanager.Status.JVM.Memory.Direct.MemoryUsed,gauge,,byte,,The amount of memory used by the JVM for the direct buffer pool,0,flink,taskmanager direct used
+flink.taskmanager.Status.JVM.Memory.Direct.TotalCapacity,gauge,,byte,,The total capacity of all buffers in the direct buffer pool,0,flink,taskmanager direct capacity
+flink.taskmanager.Status.JVM.Memory.Mapped.Count,gauge,,,,The number of buffers in the mapped buffer pool,0,flink,taskmanager mapped count
+flink.taskmanager.Status.JVM.Memory.Mapped.MemoryUsed,gauge,,byte,,The amount of memory used by the JVM for the mapped buffer pool,0,flink,taskmanager mapped used
+flink.taskmanager.Status.JVM.Memory.Mapped.TotalCapacity,gauge,,byte,,The number of buffers in the mapped buffer pool,0,flink,taskmanager mapped capacity
+flink.jobmanager.Status.JVM.Threads.Count,gauge,,,,The total number of live threads,0,flink,jobmanager live threads
+flink.taskmanager.Status.JVM.Threads.Count,gauge,,,,The total number of live threads,0,flink,taskmanager live threads
+flink.jobmanager.Status.JVM.ClassLoader.ClassesLoaded,gauge,,,,The total number of classes loaded since the start of the JVM,0,flink,jobmanager classes loaded
+flink.jobmanager.Status.JVM.ClassLoader.ClassesUnloaded,gauge,,,,The total number of classes unloaded since the start of the JVM,0,flink,jobmanager classes unloaded
+flink.taskmanager.Status.JVM.ClassLoader.ClassesLoaded,gauge,,,,The total number of classes loaded since the start of the JVM,0,flink,taskmanager classes loaded
+flink.taskmanager.Status.JVM.ClassLoader.ClassesUnloaded,gauge,,,,The total number of classes unloaded since the start of the JVM,0,flink,taskmanager classes unloaded
+flink.taskmanager.Status.Shuffle.Netty.AvailableMemorySegments,gauge,,,,The number of unused memory segments,0,flink,taskmanager shuffle available memory
+flink.taskmanager.Status.Shuffle.Netty.TotalMemorySegments,gauge,,,,The number of allocated memory segments,0,flink,taskmanager ahuffle allocated memory
+flink.task.Shuffle.Netty.Input.Buffers.inputQueueLength,gauge,,,,The number of queued input buffers,0,flink,task input buffers queued
+flink.task.Shuffle.Netty.Input.Buffers.inPoolUsage,gauge,,,,An estimate of the input buffers usage,0,flink,task input buffers used
+flink.task.Shuffle.Netty.Output.Buffers.outputQueueLength,gauge,,,,The number of queued output buffers,0,flink,task output buffers queued
+flink.task.Shuffle.Netty.Output.Buffers.outPoolUsage,gauge,,,,An estimate of the output buffers usage,0,flink,task output buffers used
+flink.task.Shuffle.Netty.Input.numBytesInLocal,count,,byte,,The total number of bytes this task has read from a local source,0,flink,task bytes read local
```

Comment:
Should this be count as well? Since the total number of bytes a task has read is always increasing?

Reply:
Which metric/line should be count?

Reply:
There are a few metrics with description (The total number of ...), which I think are always increasing. 
```
flink.taskmanager.Status.JVM.Memory.Direct.TotalCapacity
flink.jobmanager.Status.JVM.ClassLoader.ClassesLoaded
flink.jobmanager.Status.JVM.ClassLoader.ClassesUnloaded
flink.taskmanager.Status.JVM.ClassLoader.ClassesLoaded
flink.taskmanager.Status.JVM.ClassLoader.ClassesUnloaded
```
I may have missed a few

Reply:
Oh ok, I see. I'll run through the metrics again!

Reply:
I updated a few to be `count`s. There are a few that I am unsure about.